packages/block-library/src/search/index.php

```diff
@@ -132,8 +132,42 @@ function render_block_core_search( $attributes ) {
 			}
 		} else {
 			$button_classes[]       = 'has-icon';
+			$icon_dimensions        = '24';
+			$search_block_font_size = gutenberg_get_global_styles( array( 'typography', 'fontSize' ), array( 'block_name' => 'core/search', 'transforms' => array( 'resolve-variables' ) ) );
```

Comment:
TODO

Double check that these are converted to `wp_` in build.

I think that's the way it works.

Reply:
`It's not allowed to call the "gutenberg_get_global_styles()" function as its name matches the forbidden pattern: "/^[Gg]utenberg.*$/".`

Fair enough